RecoMuon/L2MuonSeedGenerator/src/L2MuonSeedGeneratorFromL1T.cc

```diff
@@ -41,6 +43,28 @@ using namespace std;
 using namespace edm;
 using namespace l1t;
 
+//--- Functions used in output sorting
+bool SortByL1Pt(L2MuonTrajectorySeed &A, L2MuonTrajectorySeed &B) {
```

Comment:
Method names should start with lower case letters, upper case being reserved to class names